cylc/flow/option_parsers.py

```diff
@@ -43,6 +43,12 @@
     setup_segregated_log_streams,
 )
 
+WORKFLOW_ID_ARG_DOC = ('WORKFLOW', 'Workflow ID')
+WORKFLOW_ID_MULTI_ARG_DOC = ('WORKFLOW ...', 'Workflow ID(s)')
+WORKFLOW_ID_OR_PATH_ARG_DOC = ('WORKFLOW | PATH', 'Workflow ID or path')
+WORKFLOW_ID_OR_NAME_ARG_DOC = ('WORKFLOW', 'Workflow ID or name')
```

Comment:
This should be "Workflow ID", should bump any commands that use this to `WORKFLOW_ID_ARG_DOC`.

```suggestion
```